clean_project/search_api.py

```diff
@@ -0,0 +1,45 @@
+"""
+Search a word in a file and return all the occurrences specified the line numbers
+"""
+import argparse
+from collections import defaultdict
+from flask import Flask, jsonify
```

Comment:
jsonify is a function. please import whole modules and not single functions. classes are ok to import alone, but in this case it's probably more convenient to import flask directly